docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+sidebar_label: Data Storage and Controls
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data does Cypress Cloud store?
+- What controls exist to control data sent to and viewed in Cypress Cloud?
+
+:::
+
+## Data Storage
+
+:::caution
+
+Please refer to the [Cypress Cloud Terms of Use](https://cloud.cypress.io/terms-of-use) and our [Security & Compliance](https://www.cypress.io/security/) guide for more specifics around reasonable use of Cypress Cloud, data storage and security at Cypress.
```

Comment:
What do we mean when we say "reasonable use"? I think we may be able to reword this a bit to streamline it, but I wanted to clarify the points.